bin/gush

```diff
@@ -32,43 +34,16 @@ if (version_compare($version, '1.9.1', 'lt')) {
 $adapterFactory = new Gush\Factory\AdapterFactory();
 
 $adapters = [
-    'github' => 'Gush\Adapter\GitHubFactory',
-    'github_enterprise' => 'Gush\Adapter\GitHubEnterpriseFactory',
-    'bitbucket' => 'Gush\Adapter\BitbucketFactory',
-    'gitlab' => 'Gush\Adapter\GitLabFactory',
+    'github' => [new LegacyAdapterFactory('Gush\Adapter\GitHubFactory'), 'GitHub'],
```

Comment:
The syntax is `'internal_name' => [factory-class, 'label']`.
Naming and labeling is decoupled from the Factory as Factory class can be reused for an Enterprise edition ;)